package/src/contexts/suggestionsContext/SuggestionsContext.tsx

```diff
@@ -91,51 +74,38 @@ export const SuggestionsContext = React.createContext({} as SuggestionsContextVa
 /**
  * This provider component exposes the properties stored within the SuggestionsContext.
  */
-export const SuggestionsProvider = <
-  Co extends string = DefaultCommandType,
-  Us extends UnknownType = DefaultUserType,
->({
+export const SuggestionsProvider = <Us extends UnknownType = DefaultUserType>({
   children,
   value,
-}: PropsWithChildren<{ value?: Partial<SuggestionsContextValue<Co, Us>> }>) => {
-  const [componentType, setComponentType] = useState<SuggestionComponentType<Co, Us>>('');
-  const [suggestions, setSuggestions] = useState<Suggestions<Co, Us>>();
-  const [suggestionsTitle, setSuggestionsTitle] = useState<React.ReactElement>();
+}: PropsWithChildren<{ value?: Partial<SuggestionsContextValue<Us>> }>) => {
+  const [componentType, setComponentType] = useState<SuggestionComponentType>('');
+  const [suggestions, setSuggestions] = useState<Suggestions<Us>>();
   const [suggestionsViewActive, setSuggestionsViewActive] = useState(false);
 
-  const openSuggestions = (
-    component: SuggestionComponentType<Co, Us>,
-    title?: React.ReactElement,
-  ) => {
+  const openSuggestions = (component: SuggestionComponentType) => {
     setComponentType(component);
-    setSuggestionsTitle(title);
     setSuggestionsViewActive(true);
   };
 
-  const updateSuggestions = (
-    newSuggestions: Suggestions<Co, Us>,
-    newSuggestionsTitle?: React.ReactElement,
-  ) => {
+  const updateSuggestions = (newSuggestions: Suggestions<Us>) => {
     setSuggestions(newSuggestions);
-    if (newSuggestionsTitle) {
-      setSuggestionsTitle(newSuggestionsTitle);
-    }
     setSuggestionsViewActive(!!componentType);
   };
 
   const closeSuggestions = () => {
     setComponentType('');
     setSuggestions(undefined);
-    setSuggestionsTitle(undefined);
     setSuggestionsViewActive(false);
   };
 
   const suggestionsContext = {
+    AutoCompleteSuggestionHeader: value?.AutoCompleteSuggestionHeader,
+    AutoCompleteSuggestionItem: value?.AutoCompleteSuggestionItem,
+    AutoCompleteSuggestionList: value?.AutoCompleteSuggestionList,
     closeSuggestions: value?.closeSuggestions || closeSuggestions,
```

Comment:
We currently accept props such as closeSuggestions, openSuggestions, updateSuggestions on Channel component. But I think these are quite useless props and don't provide any value. I think we can simply drop these props

Reply:
The trigger for opening, updating, and closing the suggestions won't work if we remove them. These functions are coupled inside `AutoCompleteInput.tsx`.